### ISSUE TITLE
Highlighting text a little difficult to read

### ISSUE THREAD

**Issue by jlmitch5** [McDonnellDean/atom-focus-white#6]
First of all, awesome theme and also contrib statement!

I like to code outside where glare can be an issue.  This theme is prefect for that (high contrast/white background to easily see cursor).  The one thing I'm having trouble with is that there is no highlight of text when you select it (line numbers becomes bold instead).

Would you be interested in a pr that adds highlighting?  I was thinking than on select, the highlight would be black and all underlying text white.

**Comment by mcdonnelldean** [comment]
Hey thanks!! 

I use this theme every day myself, I really need to give it more attention. What language are you using it with?

The black may be a little harsh but another option might be a solid black outline. Either way, send we what you think looks good 😀

Yeah fire off a PR we can see how it looks and go from there 😊 PR's are always welcome!

**Comment by jlmitch5** [comment]
I'm using atom for general front-end stuff mostly.  html/less/js (we use angular), occasionally make.

I have discovered some things:
1. The ui theme I was using was causing no highlight to show up.  After trying to hack this theme and realizing I couldn't see any changes (and seeing you actually were changing the highlight background color in the source), I tried the default atom theme and I think the grey highlight you have looks nice and definitely works for me.  I'll explore the source of the ui theme I was using to see if there is a fix possible on that side of things. https://atom.io/themes/atomic-ui if you are curious.
2. The way that atom highlights text is pretty interesting.  It actually places highlight divs under the text and the background is able to be set on those.  The text that is highlighted itself is not given any contextual `.select` classes in the dom.  There's a great thread about that here https://github.com/atom/atom/issues/2886

In short, this issue seems like it should be closed.  I'll let you know if I notice anything else, but it seems like I haven't found anything I don't like with this theme so far. :)

As an aside, I'm really impressed with how thorough your readme's installation and contrib guidelines are.  They were definitely helpful to me, and also seem like they would be very helpful to people not super familiar with git.  You're definitely helping to make the open source community more open!

**Comment by mcdonnelldean** [comment]
@jlmitch5 Interesting stuff, I'll give that UI theme a whirl. I'm using the built in light UI theme but I do want to create a high contrast complementary UI theme to pair with focus-white. 

Thanks for the kind words, yeah I'm trying to make OSS a bit more friendly and easy for people who are new to it :D
